js/models/message_handler.js

```diff
@@ -140,6 +140,31 @@ export default class MessageHandler {
     }
   }
 
+  handleRemoveOverlay = (msg) => {
+    const overlay_names = msg["name"];
```

Comment:
nit: we should have the key in `msg` also be `overlay_names` since its more descriptive